web-ng/src/app/components/form-field-editor/form-field-editor.component.html

```diff
@@ -65,7 +65,13 @@
         (delete)="onOptionDelete(i)"
       ></app-option-editor>
     </div>
-    <button mat-button type="button" (click)="onAddOption()" class="add-option">
+    <button
+      mat-button
+      type="button"
+      (click)="onAddOption()"
+      color="primary"
```

Comment:
Can we put the color setting in scss?

Reply:
The color is defined on the component by design; it pulls in the theme colors (higher level) without requiring tweaking individual colors (lower level). This is a feature, not a bug :)

Reply:
Yes, I appreciate having themes over magic numbers here and there. I meant to have:

.add-option {
  margin-left: 30px;	  
  color: primary;
}

Does that work? Or you have to import `_variables.scss` in the component scss to do so?

Reply:
Unfortunately it wouldn't. You'd have to import `_variables.scss` and use `mat-color($primary)`. There's no reason to tweak individual DOM elements since the Material Angular component is already integrated with themes:
https://material.angular.io/components/button/overview

![image](https://user-images.githubusercontent.com/228050/87204849-f1de7f00-c2d3-11ea-872f-8ec5ded7f90b.png)

e.g.:
```html
    <button mat-stroked-button>Basic</button>
    <button mat-stroked-button color="primary">Primary</button>
    <button mat-stroked-button color="accent">Accent</button>
    <button mat-stroked-button color="warn">Warn</button>
    <button mat-stroked-button disabled>Disabled</button>
```